src/main/java/com/mraof/minestuck/data/AspectTreeBlocksData.java

```diff
@@ -526,15 +788,61 @@ public static void addModels(MSBlockStateProvider provider)
 						texture("void_aspect_planks")));
 		provider.simpleHorizontal(VOID_ASPECT_LADDER, provider::ladder);
 		provider.flatItem(VOID_ASPECT_LADDER_ITEM, MSBlockStateProvider::texture);
+		provider.signBlock(AspectTreeBlocks.VOID_ASPECT_SIGN.get(), AspectTreeBlocks.VOID_ASPECT_WALL_SIGN.get(),
+				provider.blockTexture(AspectTreeBlocks.VOID_ASPECT_PLANKS.get()));
+		provider.hangingSignBlock(AspectTreeBlocks.VOID_ASPECT_HANGING_SIGN.get(), AspectTreeBlocks.VOID_ASPECT_WALL_HANGING_SIGN.get(),
+				provider.blockTexture(AspectTreeBlocks.VOID_ASPECT_PLANKS.get()));
+	}
+	
+	public static void addItemModels(MinestuckItemModelProvider provider) {
```

Comment:
These need not be done with the `MinestuckItemModelProvider`.
With `MSBlockStateProvider`, you can do a `provider.flatItem(<BLOCK>, MSBlockStateProvider::itemTexture);` for the same effect. See model generation for the skaia portal for an example.

With that, I'd like you to move these item model generations next to the block model generations for the same block.